kolibri/utils/drf_utils.py

```diff
@@ -0,0 +1,73 @@
+from functools import wraps
```

Comment:
Oh.. the only thing I can think of is whether we want to put it here or somewhere else? All our current DRF/API related machinery is in `kolibri/core` but I don't have a principled reason for it being in one place vs another.

Reply:
I don't have any preference, if you think moving the fileto `kolibri/core/utils` makes more sense, I'm happy with it

Reply:
no problem, moved